pkg/network/server.go

```diff
@@ -1142,8 +1142,11 @@ txloop:
 					}
 				}
 			}
-			if s.verifyAndPoolTX(tx) == nil {
+			err := s.verifyAndPoolTX(tx)
+			if err == nil {
 				s.broadcastTX(tx, nil)
+			} else {
+				s.log.Debug("tx handler", zap.Error(err), zap.String("hash", tx.Hash().StringLE()))
```

Comment:
How about adding the same log for `RelayTxn`?

Reply:
It's very much different in that caller receives an error and handles it the way it wants to (like RPC server returning this error to the client), so this never goes unnoticed, unlike P2P communication which is the main target of this PR.